### ISSUE TITLE
Changed import of `@yamada-ui/lucide` in Document to `*Icon`.

### ISSUE THREAD

**Issue by illionillion** [yamada-ui/yamada-ui#3409]
### Subject

Changed import of `@yamada-ui/lucide` in Document to `*Icon`.

### Description

Correct the writing style deprecated with this PR.
https://github.com/yamada-ui/yamada-ui/pull/3358

```diff
- import { Ghost } from "@yamada-ui/lucide"
+ import { GhostIcon } from "@yamada-ui/lucide"
```

Fix the code in the mdx as well as the components.

### Are you willing to participate in creating this documentation page and create a pull request?

None

**Comment by sarthakvijayvargiya** [comment]
Can I work on this

**Comment by illionillion** [comment]
@sarthakvijayvargiya Yes, please!

**Comment by hirotomoyamada** [comment]
@sarthakvijayvargiya

Hi, Thanks for the working on this issue!

We will return this issue assignment to initial state.
If you are working on this issue and would like to be reassigned, please contact us.

**Comment by hirotomoyamada** [comment]
Fixed #3611
